pkg/api/v2/types.go

```diff
@@ -398,30 +398,30 @@ type ExternalClusterState string
 
 const (
 	// PROVISIONING state indicates the cluster is being created.
-	PROVISIONING ExternalClusterState = "PROVISIONING"
+	PROVISIONING ExternalClusterState = "Provisioning"
```

Comment:
it's not compatible with the UI. The frontend will stop working

Reply:
will talk sig-ui, if something can be done